source/troubleshooting.txt

```diff
@@ -89,8 +89,59 @@ to create new projects.
 
 The user's IP address or API key that you use to authenticate is not on
 the :atlas:`access list </configure-api-access>` for the requested
-project. Add your IP address or API key to the
-:atlas:`access list </configure-api-access>` to run commands.
+project. Add your IP address or API key to the :atlas:`access list 
```

Comment:
This is not correct as you don't add the API key to the atlas access list but you add your IP to the API key access list.